src/main/python/rlbot/setup_manager.py

```diff
@@ -49,7 +49,7 @@ def setup_manager_context():
     try:
         yield setup_manager
     finally:
-        setup_manager.shut_down()
+        setup_manager.shut_down(kill_all_pids=True)
```

Comment:
did you test this to make sure that in normal use this does not cause any issues?

Reply:
I've tested it with RLBotGUI as well as RLBotTraining.
The normal (GUI) way of running RLBot does not use this contextmanager at the moment.